code/datums/datumvars.dm

```diff
@@ -243,6 +243,11 @@ datum/proc/on_varedit(modified_var) //called whenever a var is edited
 	body += "<option value='?_src_=vars;proc_call=\ref[D]'>Call Proc</option>"
 	if(ismob(D))
 		body += "<option value='?_src_=vars;mob_player_panel=\ref[D]'>Show player panel</option>"
+	if(ismob(D) || isobj(D))

```

Comment:
This should be a check for atom/movable, not mob or obj, same thing, but makes it only 1 check.